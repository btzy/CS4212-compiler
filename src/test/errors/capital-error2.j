class Main {

Void main(){
    Int x; int y; Int z;
    x=1;
}

}

class abc {

}

