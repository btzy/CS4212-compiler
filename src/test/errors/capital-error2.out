Syntax error identifier( line 4 col 15 - line 4 col 16 ) :
4 |     Int x; int y; Int z;
  |                ^
instead expected token classes are [NE, LT, LE, GT, GE, PLUS, MINUS, TIMES, DIVIDE, DOT, CONJUNCTION, DISJUNCTION, LPAREN]
Syntax error identifier( line 10 col 6 - line 10 col 9 ) :
10 | class abc {
   |       ^^^
instead expected token classes are [CLASSNAME]
Parse aborted due to above error.  To skip over error, try using lenient mode ('-lenient').
